src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -14,7 +13,7 @@
   "Schema defining the supported values for the JRuby CompileMode setting."
   (apply schema/enum supported-jruby-compile-modes))
 
-(def JRubyPuppetConfig
+(def CombinedJRubyPuppetConfig
```

Comment:
You could `dissoc` one of them? Or maybe `JRubyPuppetConfig` shouldn't have either of them?